Danger-Swift/Tests/WeTransferPRLinterTests/WeTransferLinterTests.swift

```diff
@@ -2,24 +2,56 @@ import XCTest
 @testable import WeTransferPRLinter
 @testable import Danger
 @testable import DangerFixtures
+import Files
 
 final class WeTransferLinterTests: XCTestCase {
 
+    private var buildFolder: Folder!
+
+    override func setUp() {
+        super.setUp()
+        buildFolder = try! Folder.current.createSubfolderIfNeeded(withName: ".filesTest")
+        try! buildFolder.empty()
+    }
+
     override func tearDown() {
+        try? buildFolder.delete()
```

Comment:
Maybe we can force cast here just like I the `setup` method to make sure the build folder actually gets deleted. We could also use `XCTAssertNoThrow` to make sure the test do not crash.